<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuralNetWidget</class>
 <widget class="QWidget" name="NeuralNetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="PlayButton">
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="PlayerColor">
     <item>
      <widget class="QLabel" name="PlayerColorLabel">
       <property name="text">
        <string>PlayerColor:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="PlayerColor1Radio">
       <property name="text">
        <string>PlayerColor1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">PlayerColorButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="PlayerColor2Radio">
       <property name="text">
        <string>PlayerColor2</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">PlayerColorButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="CountMCTS">
     <item>
      <widget class="QLabel" name="MinMaxDepthLabel">
       <property name="text">
        <string>Count MCTS</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLineEdit" name="MCTSCountInput">
       <property name="text">
        <string>1000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Device">
     <item>
      <widget class="QLabel" name="DeviceLabel">
       <property name="text">
        <string>Device:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="CPURadio">
       <property name="text">
        <string>CPU</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">DeviceButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="GPURadio">
       <property name="text">
        <string>GPU</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">DeviceButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Randomize">
     <item>
      <widget class="QLabel" name="RandomizeLabel">
       <property name="text">
        <string>Randomize Moves:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RandomizeYesRadio">
       <property name="text">
        <string>Yes</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">RandomizeButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RandomizeNoRadio">
       <property name="text">
        <string>No</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">RandomizeButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="NeuralNetSelect">
     <item>
      <widget class="QLabel" name="NeuralNetSelectionLabel">
       <property name="text">
        <string>NeuralNetSelection:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NetNameInput">
       <property name="text">
        <string>start</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="DeviceButtonGroup"/>
  <buttongroup name="PlayerColorButtonGroup"/>
  <buttongroup name="RandomizeButtonGroup"/>
 </buttongroups>
</ui>
